build.gradle

```diff
@@ -12,6 +12,7 @@ buildscript {
     // The buildscript {} block is odd: even though we applied dependencies.gradle above, the repositories therein
     // do not get included here. Instead, we must explicitly define the repos again. Yay for duplication.
     repositories {
+        // mavenLocal()
```

Comment:
Turns out we don't need to define mavenLocal() here to use the local version of netCDF-java artifacts, so we can just remove this one all together (we only need the one in `gradle/any/dependencies.gradle`)